src/cffi-clasp.lisp

```diff
@@ -137,13 +137,13 @@ SIZE-VAR is supplied, it will be bound to SIZE during BODY."
 (defun make-shareable-byte-vector (size)
   "Create a Lisp vector of SIZE bytes that can passed to
 WITH-POINTER-TO-VECTOR-DATA."
-  (make-array size :element-type '(unsigned-byte 8)))
-
-;; frgo, 2016-07-02: TODO: Implemenent!
-;; (defmacro with-pointer-to-vector-data ((ptr-var vector) &body body)
-;;   "Bind PTR-VAR to a foreign pointer to the data in VECTOR."
-;;   `(let ((,ptr-var (si:make-foreign-data-from-array ,vector)))
-;;      ,@body))
+  ;; borrowed from static.vectors
```

Comment:
What's `static.vectors`?

Reply:
See https://github.com/clasp-developers/static-vectors for clasp or
https://github.com/sionescu/static-vectors

Basically this allows an array containing e.g. simple bytes to be passed to ffi-world (with the guarantee that the gc won't move the memory).
With-pointer-to-vector-data and make-shareable-byte-vector do very similar things, so I borrowed from clasp implementations of static-vectors

Reply:
Oh, OK. I think you can drop those comments. @sionescu won't mind. :-)